theme/instructor-menu.js

```diff
@@ -0,0 +1,71 @@
+(function handleInstructor() {
+  function handleInstructorMenu() {
```

Comment:
Did you discuss simply adding this to the theme instead? Since this will be a permanent thing on all pages, it should go into [index.hbs](https://github.com/google/comprehensive-rust/blob/main/theme/index.hbs) or [head.hbs](https://github.com/google/comprehensive-rust/blob/main/theme/head.hbs).

That would simplify the code a lot since you can just write out the HTML directly.

Reply:
Why was this not done for the red box, you might ask? Because that was not supposed to be rendered for people looking at the course in the published version.